src/core/internal/fetch/fetchClient.ts

```diff
@@ -27,24 +32,19 @@ export class FetchClient implements HttpClient {
         return this.request(url, 'PATCH', config, data);
     }
 
-    private async request(url: string, method: string, config?: { headers: any }, data?: any) {
-        let body = undefined;
-        if (!!data) {
-            if (typeof data === 'object') {
-                body = JSON.stringify(data);
-                config = config || { headers: {} };
-                config.headers = config.headers || {};
-                config.headers['Content-Type'] = 'application/json';
-            } else {
-                body = data;
-            }
-        }
-        await this.waitInQueue();
+    private async request(url: string, method: string, config?: IRequestConfig, data?: any) {
+        const reqConfig = config || {};
+        const reqData = data || {};
```

Comment:
Also unsafe thing. For some calls we do not need to send any body in request. This code breaks this. Just keep in mind :
If we have body then attach in to request. If body is object then convert it by using `JSON.stringify` function and make sure that 'Content-Type' is 'application/json' (only if body != null and body is object). And no further changes in httpd config.